src/xcode/ENA/ENA/Source/Client/HTTP Client/CachingHTTPClient.swift

```diff
@@ -173,6 +186,44 @@ class CachingHTTPClient: AppConfigurationFetching, StatisticsFetching, QRCodePos
 			}
 		}
 	}
+
+	// MARK: LocalStatisticsFetching
+	
+	/// Fetches local statistics
+	/// - Parameters:
+	///   - administrativeUnit: string to pass administrative unit to the API to get local statistics
+	///   - etag: an optional ETag to download only versions that differ the given tag
+	///   - completion: result handler
+	func fetchLocalStatistics(
+		administrativeUnit: String,
```

Comment:
I would suggest we create a `typealias` for *administrativeUnit* to simplify function calls later:
```
typealias AdministrativeUnit = String
```
This should prevent devs from putting random strings into functions like this.